udata/templates/territories/territory.html

```diff
@@ -46,5 +46,5 @@
 {% block extra_js %}
     <script src="{{ static('vue-common.js') }}"></script>
     <script src="{{ static('territory.js') }}"></script>
-    <script src="{{ static('widgets.js', _burst=False) }}" id="udata" async defer onload="udataScript.load()"></script>
+    <script src="{{ static('widgets.js', _burst=False) }}" id="udata" async defer onload="udataScript.loadDatasets()"></script>
```

Comment:
Yes, and that's what is done here: https://github.com/opendatateam/udata/pull/496/files/4e22e4e8bddb70cd7b81a37aba7581a1430f93b6#diff-963e0f551bd557273268a73a942c30e7R168